core/src/mindustry/input/MobileInput.java

```diff
@@ -424,7 +424,7 @@ public boolean isBreaking(){
     @Override
     public void useSchematic(Schematic schem){
         selectRequests.clear();
-        selectRequests.addAll(schematics.toRequests(schem, player.tileX(), player.tileY()));
+        selectRequests.addAll(schematics.toRequests(schem, (int)(Core.camera.position.x / tilesize), (int)(Core.camera.position.y / tilesize)));
```

Comment:
This still isnt using the appropriate method to convert world coords to tile coords, use `World.toTile` instead of dividing and then casting to an int